src/convertToObject.js

```diff
@@ -12,7 +12,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const solutionArr = sourceString.split(';'); // get a split array
+  const solutionObj = {}; // object for getting solution
+  for (let i = 0; i < solutionArr.length; i++) {
```

Comment:
new line before for loop

Reply:
New empty line is added:
![image](https://user-images.githubusercontent.com/32244670/63268073-079e1c80-c29c-11e9-9d01-7e2353297946.png)